__tests__/index.spec.ts

```diff
@@ -1,17 +1,23 @@
 import * as path from 'path';
 import * as bodyParser from 'koa-bodyparser';
+import { ChowOptions } from 'oas3-chow-chow';
+import * as koa from 'koa';
 
 import { oas } from '../src';
 import { createContext } from './helpers/createContext';
-import { ChowOptions } from 'oas3-chow-chow';
 
-describe('Koa Oas3', () => {
-  const mw = oas({
-    file: path.resolve('./__tests__/fixtures/pet-store.json'),
-    endpoint: '/openapi',
-    uiEndpoint: '/openapi.html',
-    validatePaths: ['/pets']
-  });
+describe('Koa Oas3', async () => {
```

Comment:
You don't need async here, jest is complaining too: https://travis-ci.org/github/atlassian/koa-oas3/builds/714344155